M/MAGEMin/build_tarballs.jl

```diff
@@ -34,11 +34,16 @@ INC="-I${includedir}"
 # Compile library:
 make -j${nproc} CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" lib
 
-# compile binary
+# Compile binary
 make -j${nproc} CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" all
 
+# Windows sometimes adds an extension to the binary and sometimes not...
+if test -f "MAGEMin.exe"; then
+    mv MAGEMin.exe MAGEMin
+fi
+
```

Comment:
Isn't there a way to set the name of the executable on the command line, instead of doing this?

Reply:
Done (and passes the tests now)